sql/character.anticheat.sql

```diff
@@ -45,6 +45,18 @@ CREATE TABLE IF NOT EXISTS `players_reports_status` (
   PRIMARY KEY (`guid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+-- Dumping structure for table characters.lua_cheaters
+DROP TABLE IF EXISTS `lua_cheaters`;
+CREATE TABLE IF NOT EXISTS `lua_cheaters` (
+  `guid` int(11) NOT NULL,
+  `account` int(11) NOT NULL,
+  `macro` varchar(255) DEFAULT NULL,
+    PRIMARY KEY (`guid`,`account`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- Dumping data for table characters.lua_cheaters: ~0 rows (approximately)
+DELETE FROM `lua_cheaters`;
```

Comment:
Please remove this as we are creating the table from scratch

Reply:
https://github.com/TrinityCore/TrinityCoreCustomChanges/commit/1d1cdda4d4a033b99dde03f0de4606956be0f989